cranelift-wasm/Cargo.toml

```diff
@@ -20,6 +20,9 @@ failure = { version = "0.1.1", default-features = false, features = ["derive"] }
 failure_derive = { version = "0.1.1", default-features = false }
 log = { version = "0.4.6", default-features = false }
 cast = { version = "0.2.2", default-features = false }
+target-lexicon = { version = "0.3.0", default-features = false }
+gimli = "0.17.0"
+faerie = "0.9.1"
```

Comment:
gimli and especially faerie are fairly heavyweight dependencies that not all cranelift-wasm users will want. Would you mind either putting these under control of a feature, or (at the risk of suggesting yet another crate change) moving this code into a seperate crate, maybe cranelift-wasmdebug?

Reply:
added a feature